src/core/reducers/errorPage.js

```diff
@@ -0,0 +1,31 @@
+import {
+  REDUX_CONNECT_END_GLOBAL_LOAD,
+  REDUX_CONNECT_LOAD_FAIL,
+} from 'core/constants';
+
+export const initialState = {
+  clearOnNext: false,
+  error: null,
+  hasError: false,
+  statusCode: null,
+};
+
+export default function errorPage(state = initialState, action) {
+  const { payload } = action;
+  switch (action.type) {
+    case REDUX_CONNECT_END_GLOBAL_LOAD:
+      if (state.clearOnNext) {
+        return { ...state, ...initialState };
+      }
+      return { ...state, clearOnNext: true };
+    case REDUX_CONNECT_LOAD_FAIL:
+      return {
+        ...state,
+        error: payload.error,
+        hasError: true,
+        statusCode: payload.error.response.status,
```

Comment:
Since redux-connect will set `error` to anything that gets thrown (by, for example, `callApi()`) then you can't rely on this having the property `response`. This needs to be defensive otherwise it dies saying "error does not have a response property" or something.

I found it while testing which led me to https://github.com/mozilla/addons-frontend/issues/1701 (but that's completely unrelated!)